Unable to publish @amityco/react-native-formdata-polyfill v1.0.1 to NPM.

# Bug Report

[it's a release related thing](https://github.com/AmityCo/react-native-formdata-polyfill/runs/4897190962?check_suite_focus=true)

<img width="1421" alt="Screen Shot 2022-01-21 at 9 00 44 PM" src="https://user-images.githubusercontent.com/6318491/150539763-7728c35c-abb8-47ed-9293-9d824034a6e4.png">

```
npm ERR! code E402
npm ERR! 402 Payment Required - PUT https://registry.npmjs.org/@amityco%2freact-native-formdata-polyfill - You must sign up for private packages

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-01-21T13_53_23_695Z-debug.log
Error: Error: Unable to publish @amityco/react-native-formdata-polyfill v1.0.1 to NPM. 
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -20,3 +20,4 @@ jobs:
       - uses: JS-DevTools/npm-publish@v1
         with:
           token: ${{ secrets.NPM_TOKEN }}
+          access: public

